### PR TITLE
Add 'Request Localization Sample' key to the resource files

### PR DESCRIPTION
**Pull request by hishamco**

### CODE REVIEW

**Comment by Eilon** [comment]
These are more churn than makes sense for this repo. Please log issues for discussion before submitting PRs so we can discuss the best approach.

File: samples/LocalizationSample/My/Resources/Startup.zh.resx [hunk 1]
```diff
@@ -117,7 +117,7 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="Hello" xml:space="preserve">
-    <value>您好</value>
+  <data name="Request Localization Sample" xml:space="preserve">
+    <value>申请本地化样品</value>
```
Comment (Kagamine) [inline samples/LocalizationSample/My/Resources/Startup.zh.resx]: It should be "本地化请求样例"

**Comment by hishamco** [comment]
@Kagamine you can make a PR for this, while, this has been closed

**Comment by Kagamine** [comment]
OK~

**Comment by hishamco** [comment]
Nope, just change the value if it's wrong

**Pull request closed without merging**